lib/PayPal/Auth/OAuthTokenCredential.php

```diff
@@ -267,6 +286,11 @@ private function generateAccessToken($config, $refreshToken = null)
             $params['grant_type'] = 'refresh_token';
             $params['refresh_token'] = $refreshToken;
         }
+
+        if ($this->subject != null && $refreshToken != null) {
```

Comment:
If `subject` is associated with a `refreshToken`, should we just pass the `subject` as an argument here, instead of adding it to a constructor ?